src/test/java/org/folio/rest/impl/FeeFineReportsAPITest.java

```diff
@@ -189,14 +192,12 @@ public void emptyReportWhenRefundedAfterEndDate() {
 
   @Test
   public void badRequestWhenParameterIsMissingOrMalformed() {
-    refundReportsClient.getByParameters("nonExistingParam=1", HTTP_BAD_REQUEST);
-    refundReportsClient.getByParameters("startDate=2020-01-01", HTTP_BAD_REQUEST);
-    refundReportsClient.getByParameters("endDate=2020-01-01", HTTP_BAD_REQUEST);
-    refundReportsClient.getByParameters("startDate=not-a-date", HTTP_BAD_REQUEST);
-    refundReportsClient.getByParameters("endDate=not-a-date", HTTP_BAD_REQUEST);
-    refundReportsClient.getByParameters("startDate=2020-01-01&endDate=not-a-date", HTTP_BAD_REQUEST);
-    refundReportsClient.getByParameters("startDate=not-a-date&endDate=2020-01-01", HTTP_BAD_REQUEST);
-    refundReportsClient.getByParameters("startDate=not-a-date&endDate=not-a-date", HTTP_BAD_REQUEST);
+    refundReportsClient.getFeeFineRefundReports("nonExistingParam=1", "2020-01-01", HTTP_UNPROCESSABLE_ENTITY);
```

Comment:
I think it's arguable that the server should return 400 when it receives malformed arguments such as these.

Reply:
But I guess if the body was built correctly and there is only semantical error for startDate value is should be 422... Am I wrong?

Reply:
I have renamed this test and removed some redundant cases

Reply:
After removing the invalid syntax cases from this test there is no reason to return a 400 here.